templates/dashboard/alias_transfer_send.html

```diff
@@ -29,18 +25,40 @@ <h1 class="h3">Transfer {{ alias.email }}</h1>
           {{ alias_transfer_url }}
         </em>
 
+        <p class="mt-5">
+            Please copy the transfer URL. <strong>We won't be able to display it again</strong>. If you need to access it again you can generate a new URL.
+        </p>
+
+        <p class="mt-2">
+            This transfer URL is <strong>valid for 30 minutes</strong>. If it hasn't been used by then it will be automatically disabled.
+        </p>
+
         <form method="post">
           <input type="hidden" name="form-name" value="remove">
-          <button class="btn btn-warning mt-2">Remove Share URL</button>
+          <button class="btn btn-warning mt-2">Remove alias transfer URL</button>
           <div class="small-text">
             If you don't want to share this alias anymore, you can remove the share URL.
           </div>
         </form>
       {% else %}
-        <form method="post">
-          <input type="hidden" name="form-name" value="create">
-          <button class="btn btn-primary mt-2">Create Share URL</button>
-        </form>
+        {% if link_active %}
+          <p class="mt-2">
```

Comment:
I'd add "alert alert-info" class here so user can focus on this info.

Reply:
Updated.